packages/routers/src/StackRouter.tsx

```diff
@@ -209,6 +215,7 @@ export function StackRouter(options: StackRouterOptions) {
         index: routes.length - 1,
         routeNames,
         routes,
+        preloadedRoutes: [], // ?
```

Comment:
it is actually possible to have some things before rehydration. Screen names should be valid and keys matching (if the are already in the stack)